error when removing a label that a story doesn't have

If you try to run `cr` on a story that's already in review, it will error when trying to remove the `in_progress` label:

```
panic: DELETE https://api.github.com/repos/conjurinc/dev-flow/issues/8/labels/in%20progress: 500  []
```

We should either check for a tag before performing the deletion or possibly just hide the error.

--- a/cmd/codereview.go
+++ b/cmd/codereview.go
@@ -28,18 +28,17 @@ var codereviewCmd = &cobra.Command{
 		issueKey := issuetracking.GetIssueKeyFromBranchName(branchName)
 		issue := it.Issue(issueKey)
 
-		progressLabelName := viper.Get("labels.in_progress")
+		progressLabelName := viper.GetString("labels.in_progress")
 
-		if progressLabelName != nil {
-			it.RemoveIssueLabel(issue, progressLabelName.(string))
-
+		if progressLabelName != "" && issue.HasLabel(progressLabelName) {
+			it.RemoveIssueLabel(issue, progressLabelName)
 			fmt.Printf("Removed label '%v' from issue %v.\n", progressLabelName, *issue.Number)
 		}
 		
-		reviewLabelName := viper.Get("labels.in_review")
+		reviewLabelName := viper.GetString("labels.in_review")
 
-		if reviewLabelName != nil {
-			err := it.AddIssueLabel(issue, reviewLabelName.(string))
+		if reviewLabelName != "" {
+			err := it.AddIssueLabel(issue, reviewLabelName)
 
 			if err != nil {
 				fmt.Println(err)

--- a/cmd/complete.go
+++ b/cmd/complete.go
@@ -16,8 +16,7 @@ import (
 var completeCmd = &cobra.Command{
 	Use:   "complete",
 	Short: "Merges the story branch and completes the issue.",
-	Run: func(cmd *cobra.Command, args []string) {
-
+	Run: func(cmd *cobra.Command, args []string) {		
 		vc := versioncontrol.GetClient()
 		branchName := vc.CurrentBranch()
 
@@ -54,11 +53,10 @@ var completeCmd = &cobra.Command{
 
 		it.AssignIssue(issue, pr.Creator)
 
-		reviewLabelName := viper.Get("labels.in_review")
+		reviewLabelName := viper.GetString("labels.in_review")
 		
-		if reviewLabelName != nil {
-			it.RemoveIssueLabel(issue, reviewLabelName.(string))
-
+		if reviewLabelName != "" && issue.HasLabel(reviewLabelName) {
+			it.RemoveIssueLabel(issue, reviewLabelName)
 			fmt.Printf("Removed label '%v' from issue %v.\n", reviewLabelName, *issue.Number)
 		}
 

--- a/cmd/start.go
+++ b/cmd/start.go
@@ -25,10 +25,10 @@ var startCmd = &cobra.Command{
 		it.AssignIssue(issue, user)
 		fmt.Printf("Assigned issue %v to user %v.\n", *issue.Number, user)
 
-		progressLabelName := viper.Get("labels.in_progress")
+		progressLabelName := viper.GetString("labels.in_progress")
 
-		if progressLabelName != nil {
-			err := it.AddIssueLabel(issue, progressLabelName.(string))
+		if progressLabelName != "" {
+			err := it.AddIssueLabel(issue, progressLabelName)
 
 			if err != nil {
 				fmt.Println(err)

--- a/common/common.go
+++ b/common/common.go
@@ -30,3 +30,12 @@ func (issue Issue) BranchName() string {
 	
 	return fmt.Sprintf("%v--%v", *issue.Number, title)
 }
+
+func (issue Issue) HasLabel(label string) bool {
+	for _, issueLabel := range issue.Labels {
+		if issueLabel == label {
+			return true
+		}
+	}
+	return false
+}

